modules/uavcan

```diff
@@ -1 +1 @@
-Subproject commit 6dd432c9742c22e1dd1638c7f91cf937e4bdb2f1
+Subproject commit 7ce96d6c1e5aeb82fa90ade26a49552c122a984b
```

Comment:
I dont understand submodules. I had difficulties getting it to build so copied the files from a clean build but git noted these files changed so I commited them . Maybe someone else can comment on that?